### PR TITLE
Mm 44852 request an update

### PR DESCRIPTION
**Pull request by gbochora**
#### Summary
This PR adds `Request update` functionality.
Change-log:
- Add `request update` API endpoint
- Add `Request update` button implementation
- Add toast message on success/failure
- Fix the `success` toast text color. It was the same as the background color, and the text was invisible.

https://user-images.githubusercontent.com/4368372/176995678-bd51637b-e82b-4332-934a-52ae9c58bdba.mp4

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-44852

#### Checklist

~~- [ ] Telemetry updated~~
~~- [ ] Gated by experimental feature flag~~
- [x] Unit tests updated

### CODE REVIEW

File: client/playbook_runs.go [hunk 1]
```diff
@@ -160,6 +160,21 @@ func (s *PlaybookRunService) UpdateStatus(ctx context.Context, playbookRunID str
 	return nil
 }
 
+func (s *PlaybookRunService) RequestUpdate(ctx context.Context, playbookRunID, userID string) error {
```
Comment (agarciamontoro) [inline client/playbook_runs.go:163]: The userID is not needed here, right?
```suggestion
func (s *PlaybookRunService) RequestUpdate(ctx context.Context, playbookRunID) error {
```

File: webapp/src/components/backstage/toast_banner.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ const StyledText = styled.div<{toastType: ToastType}>`
     text-align: center;
 
     margin: 0px 8px;
-    color: ${({toastType}) => (toastType === ToastType.Failure ? 'var(--center-channel-bg)' : 'var(--center-channel-color)')};
+    color: var(--center-channel-bg);
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/toast_banner.tsx:61]: Whoops, this was my mistake, sorry!

**Review by agarciamontoro — APPROVED** [review]
Looks great, @gbochora! :tada: There's just a non-blocking comment on the `client` package. Thanks!

File: client/playbook_runs.go (hunk 1 above)
Comment (agarciamontoro) [reply client/playbook_runs.go:163]: I thought this would have been caught by the linter :thinking:

**Comment by agarciamontoro** [comment]
This fixes https://mattermost.atlassian.net/browse/MM-45418 too.

**Review by trilopin — CHANGES_REQUESTED** [review]
Check [message in community](https://community-daily.mattermost.com/core/pl/qoowkqw1utb65cpw73rt1mzyhc) . This "request update" should be linked to professional license, having upgrade banner otherwise.

**Review by trilopin — APPROVED** [review]
Code is fine, missing license check will be added in a future PR.

**Pull request merged**